pom.xml

```diff
@@ -1959,6 +1959,34 @@ under the License.
 
 							<removeUnusedImports />
 						</java>
+						<scala>
+							<scalafmt>
+								<version>3.4.3</version>
+								<file>.scalafmt.conf</file>
+							</scalafmt>
+							<!-- license header is handled separately directly by spotless -->
+							<licenseHeader>
+								<content>/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
```

Comment:
isn't this covered by the rat-plugin?
Does it automatically add the header when doing spotless:apply? If so, is this only supported for Scala?